e2e-tests/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
A better solution would be to upgrade cypress version, I guess better do it in separate PR.